index.js

```diff
@@ -2,14 +2,19 @@
 const ansiEscapes = require('ansi-escapes');
 const supportsHyperlinks = require('supports-hyperlinks');
 
-module.exports = (text, url, options = {}) => {
-	if (!supportsHyperlinks.stdout) {
+const terminalLink = (text, url, options = {}, target = 'stdout') => {
```

Comment:
Put the `target` into `options` will be better.

Reply:
Resolved this by doing a shorthand property assignment: `{target = 'stdout', ...options} = {}`.

I assumed this syntax is OK as Travis is running Node 10 & 12.